ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py

```diff
@@ -114,12 +114,23 @@ def config_key(cls, datasource, context):
     @classmethod
     def params(cls, datasource, context):
         te = lambda x: datasource.talesEval(x, context)
+        query = ""
+        query_error = True
+
+        if datasource.query.strip() != '$':
```

Comment:
I don't understand the point of this conditional. When would the query ever be the literal '$'?